docs/source/adding_new_benchmark.rst

```diff
@@ -0,0 +1,180 @@
+               device: QuantumDevice
+           ) -> NewBenchmarkData:
+               # TODO: Implement logic for dispatching the job
+               pass
+
+           def poll_handler(
+               self,
+               job_data: BenchmarkData,
+               result_data: list[GateModelResultData]
+           ) -> NewBenchmarkResult:
+               # TODO: Implement logic for retrieving and processing results
+               pass
+
+Defining the Schema
+*******************
+
+To standardize and validate the input parameters for each benchmark, **metriq-gym** uses JSON Schema. Add a new
+file named :file:`new_benchmark.schema.json` to the :file:`schemas/` directory:
+
+.. code-block:: json
+
+    {
+        "$id": "metriq-gym/new_benchmark.schema.json",
+        "$schema": "https://json-schema.org/draft/2020-12/schema",
+        "title": "New Benchmark",
+        "description": "Schema definition for New Benchmark, describing its configurable parameters.",
+        "type": "object",
+        "properties": {
+            "benchmark_name": {
+                "type": "string",
+                "const": "New Benchmark",
+                "description": "Name of the benchmark. Must be 'New Benchmark' for this schema."
+            },
+            "num_qubits": {
+                "type": "integer",
+                "description": "Number of qubits to be used in the circuit(s).",
+                "minimum": 1,
+                "examples": [5]
+            },
+            "shots": {
+                "type": "integer",
+                "description": "Number of measurement shots (repetitions) to use when running the benchmark.",
+                "default": 1000,
+                "minimum": 1,
+                "examples": [1000]
+            },
+            "...": {
+                "description": "Placeholder for additional properties as needed."
+            }
+        },
+        "required": ["benchmark_name", "num_qubits"]
+    }
+
+This schema ensures that any job payload for the new benchmark meets the required format and constraints.
+
+Example Schema
+**************
+
+Provide a sample JSON file demonstrating how to supply parameters for this benchmark. Place this file in
+:file:`schemas/examples/new_benchmark.example.json`:
+
+.. code-block:: json
+
+    {
+        "benchmark_name": "New Benchmark",
+        "num_qubits": 5,
+        "shots": 1000
+    }
+
+This file offers a reference for developers and users on how to structure the JSON payload for your new benchmark.
+
+Registering the New Benchmark
+*****************************
+
+1. **Add to job_type.py**
+
+   Open the :file:`metriq_gym/job_type.py` file and register your new benchmark in the :code:`JobType` enumeration:
+
+   .. code-block:: python
+
+       from metriq_gym.job_type import JobType
+       from enum import StrEnum
+
+       class JobType(StrEnum):
+           NEW_BENCHMARK = "New Benchmark"
+           ...
+
+2. **Initialize in benchmarks/__init__.py**
```

Comment:
This makes me think that perhaps job_type.py should be in the same file as well...

Reply:
Yeah, I think this was something we had previously discussed, but we were involved in refactoring another part of the codebase and didn't want to conflate that MR with too many moving parts. In general though, I still agree with that restructuring. I opened an issue to track that here:
https://github.com/unitaryfoundation/metriq-gym/issues/294